src/components/Navbar.css

```diff
@@ -0,0 +1,60 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0.8rem 2.1rem;
+}
+
+.navbar-contents {
+  display: flex;
+  gap: 2rem;
+}
+
+.navbar-contents h3 {
+  font-size: 2rem;
+  color: #0290ff;
+}
+
+.navbar-contents ul {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.navbar-contents li {
+  list-style: none;
+  font-size: 1.4rem;
+}
+
```

Comment:
- [ ] At the moment, your design layout is not aligned as the original design. See below images for reference.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/232201885-4b2835bc-8e3b-446d-a126-3bc82433c088.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/232201895-301e02c9-d9af-42ce-9862-9b7fcf4b3556.png)